src/Microsoft.OpenApi.Hidi/Microsoft.OpenApi.Hidi.csproj

```diff
@@ -34,8 +34,8 @@
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="8.0.0" />
     <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="8.0.0" />
     <PackageReference Include="System.CommandLine" Version="2.0.0-beta4.22272.1" />
```

Comment:
I'm a bit confused? the version wasn't bumped for hidi

Reply:
Here's the PR to bump the version :)
https://github.com/microsoft/OpenAPI.NET/pull/1650